app/instrumentation-tests/src/org/commcare/androidTests/LogSubmissionTest.kt

```diff
@@ -42,4 +51,25 @@ class LogSubmissionTest: BaseTest() {
                 "Log submission was unsuccessful"
         )
     }
+
+    @Test
+    fun testReportProblem() {
+        InstrumentationUtility.selectOptionItem(withText("Advanced"))
+        onView(withText("Report Problem"))
+            .perform(click())
+        InstrumentationUtility.enterText(R.id.ReportText01,"This is a test for Report Problem")
+        InstrumentationUtility.hideKeyboard()
+        onView(withText("Submit Report")).isPresent()
+        InstrumentationUtility.rotateLeft()
+        InstrumentationUtility.matchTypedText(R.id.ReportText01,"This is a test for Report Problem")
```

Comment:
you should put the input text in a variable so that we can verify it later without repeating it.

Reply:
assigned the text to a variable and used the same.